packages/office-ui-fabric-react/src/components/Nav/Nav.tsx

```diff
@@ -144,8 +144,9 @@ export class Nav extends BaseComponent<INavProps, INavState> implements INav {
           [styles.compositeLinkIsExpanded]: link.isExpanded,
           [styles.compositeLinkIsSelected]: isLinkSelected
         }) }>
-        { (nestingLevel === 0 && link.links && link.links.length > 0 ?
```

Comment:
@dzearing Rush breaks on my system- I can only submit an answer for the first question. After that it becomes unresponsive.